Source/WebCore/dom/InlineStyleSheetOwner.cpp

```diff
@@ -145,14 +145,12 @@ void InlineStyleSheetOwner::clearSheet()
     sheet->clearOwnerNode();
 }
 
-inline bool isValidCSSContentType(Element& element, const AtomString& type)
+inline bool isValidCSSContentType(Element&, const AtomString& type)
```

Comment:
Please drop Element parameter and update call sites.

Reply:
It is used in Line 168, so I kept it.
`if (!isValidCSSContentType(element, m_contentType))`

Reply:
Yeah I agree with Chris, we should remove the element parameter from the call sites too

Reply:
`if (!isValidCSSContentType(element, m_contentType))`
->
`if (!isValidCSSContentType(m_contentType))`

Reply:
On it, just checking locally that it compiles and work etc. :-)